bpf/lib/conntrack.h

```diff
@@ -181,6 +181,16 @@ static __always_inline __u8 __ct_lookup(const void *map, struct __ctx_buff *ctx,
 	entry = map_lookup_elem(map, tuple);
 	if (entry) {
 		cilium_dbg(ctx, DBG_CT_MATCH, entry->lifetime, entry->rev_nat_index);
+
+		if (dir == CT_SERVICE && entry->tx_closing) {
+		    /* There is an existing entry for this service. However, the connection
+		     * was already closed in the past. The backend associated with this may
```

Comment:
nit: "The backend associated with this may be overloaded" doesn't sound like the right motivation for this action, maybe something like "We should allow the new connection to have a chance to pick a new backend"?

Reply:
Updated.